src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  /* font-size: 0; */
+}
+
+/* comment .stars and uncomment the others, font-size metters there which i am curios about, why?*/
+.stars {
+  display: flex;
+  flex-flow: row wrap;
+  width: 100px;
```

Comment:
do not set the width explicitly, it is not needed here.

also, `flex-flow` is a shorthand property for
- flex-direction
- flex-wrap

`flex-direction` has the default value of `row` and `flex-wrap` is not needed here. so you can just remove it